src/Umbraco.Web/PublishedContentExtensions.cs

```diff
@@ -892,7 +892,7 @@ public static IEnumerable<IPublishedContent> Children(this IPublishedContent con
         /// <returns>The children of the content, of any of the specified types.</returns>
         public static IEnumerable<IPublishedContent> ChildrenOfType(this IPublishedContent content, string contentTypeAlias, string culture = null)
         {
-            return content.Children(x => contentTypeAlias.InvariantContains(x.ContentType.Alias), culture);
+            return content.Children(x => contentTypeAlias.InvariantEquals(x.ContentType.Alias), culture);
```

Comment:
As `x.ContentType.Alias` shouldn't ever be `null`, it's better to swap this to:
```suggestion
            return content.Children(x => x.ContentType.Alias.InvariantEquals(contentTypeAlias), culture);
```
This also keeps it in the same order as the other methods 👍